opentech/apply/stream_forms/templates/stream_forms/render_field.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <h5>{{ value.field_label }}</h5>
+    <div>{% block data_display %}{{ data|default:"No response" }}{% endblock %}</div>
+</div>
```

Comment:
@chris-lawton These are the changes to the field i mentioned. See also `render_list_field` and `render_unsafe_field`